backend/graminsta/post/models.py

```diff
@@ -0,0 +1,29 @@
+"""
+Here defines the database tables for post app
+"""
+
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Post(models.Model):
+    """
+    Post Table
+    """
+    postId = models.IntegerField(default=0)
+    publisher = models.ForeignKey(User, on_delete=models.CASCADE,
+                                  related_name="Publisher_Of_Post")
+    time = models.DateTimeField(auto_now=True)
+    markedUser = models.ManyToManyField(
+        User, related_name="User_Who_Marked_Post")
+    content = models.BinaryField()
+
+
+class Comment(models.Model):
+    """
+    Comment Table
+    """
+    commentId = models.IntegerField(default=0)
+    publisher = models.ForeignKey(User, on_delete=models.CASCADE)
+    time = models.DateTimeField(auto_now=True)
+    postId = models.ForeignKey(Post, on_delete=models.CASCADE)
```

Comment:
- Django gives each model default id field. Refer to https://docs.djangoproject.com/en/3.0/topics/db/models/#automatic-primary-key-fields
- Comment-post relationship is a many-to-one relationship. I think `Post` should also be a foreign key.
- A description field should be added to store comment content.

Reply:
And just use `post = models.ForeignKey` instead of `postId`, as Django will resolve the foreignKey relationship during runtime and `some_comment.post` will give you a `Post` object instead of a primary key.